src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -227,11 +227,10 @@ Every NRQL query will begin with a `SELECT` statement or a `FROM` clause. All ot
         id="extrapolate-example-1"
         title="Example of extrapolating throughput"
       >
-        A query that will show the extrapolated throughput of a service named `interestingApplication`.
+        A query that will show the extrapolated throughput of a service named 'interestingApplication'.
 
         ```
-        SELECT count(*) FROM Transaction WHERE appName='interestingApplication' 
-        SINCE 60 minutes ago EXTRAPOLATE
+        SELECT count(*) FROM Transaction WHERE appName='interestingApplication'
```

Comment:
This does not seem related to the original PR

Reply:
You're right this was a hiccup from combing through lines of text. Fixing!